src/Compilers/CSharp/Portable/Symbols/Source/TypeParameterInfo.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Could you enable nullability for this new file?

Reply:
totally!